priv/templates/phx.gen.live/index.ex

```diff
@@ -0,0 +1,43 @@
+defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>Live.Index do
+  use <%= inspect context.web_module %>, :live_view
+
+  alias <%= inspect context.module %>
+  alias <%= inspect schema.module %>
+
+  def render(assigns) do
+    Phoenix.View.render(<%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>View, "index.html", assigns)
+  end
```

Comment:
Should we make this a convention? If you don't define a `render` clause, and the module name is `MyApp.FooLive.Index`, we will automatically call `MyApp.FooView, "index.html"`? I think it can positively reduce the amount of boilerplate.

Reply:
Note: we can remove this and colocate templates in latest LV.